server/src/main/java/com/senlainc/warsaw/tyurin/service/OrderService.java

```diff
@@ -64,6 +67,7 @@ public void shiftCompletionDateTime(long id, LocalDateTime completionDateTime) t
             orderDAO.getOrder(id).setCompletionDate(completionDateTime);
         } else {
             System.out.println("Shifting completion time was prohibited");
```

Comment:
You don't need both. Use logger and get rid of SOUTs

Reply:
Done